### PR TITLE
DRAFT: adds multisig/ledger

### PR DESCRIPTION
**Pull request by m4sterbunny**

### CODE REVIEW

File: docs/governance/technical_implementation/vote.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+
+To verify the transaction details on the Ledger device, connect your Ledger hardware wallet to the computer and open the Solana app. Approve the transaction using the physical buttons on the device.
+
+> Remember, a transaction request won't be signed until approvals are provided by other parties and the minimum threshold specified in the MultiSig wallet is met. In this example, the threshold is 3, so 2 or more signatures are required to execute this transaction.
+
+### 5.3 Execute a transaction
+
+Once the minimum threshold of signatures is collected, the transaction may be executed, and the assets will be transferred.
+
+<!-- in some multisigs there is no need to execute -- ruleset is to execute once threshold is met pls verify this is valid -->
```
Comment (m4sterbunny) [inline docs/governance/technical_implementation/vote.md:192]: @YYYurchenko can you confirm ?

File: docs/governance/technical_implementation/vote.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+
+3.2 Next, connect your Ledger wallet as per [Ledger's documentation](https://support.ledger.com/hc/en-us/articles/4408131265169-Set-up-and-use-Phantom-to-access-your-Ledger-Solana-SOL-account?docs=true)
+
+> Please make sure to remember the password you set up for Phantom.
+
+## Step 4: Access your squad on Squads
+
+4.1 To access Squads on DevNet, go to [https://devnet.squads.so/connect-squad](https://devnet.squads.so/connect-squad). Click on **Connect wallet**, then **Agree and continue** when prompted to acknowledge terms and conditions.
+
+<!-- todo Yuri thinks there should be a mainnet url -->
```
[the remark this answers is not in the document]
Comment (m4sterbunny) [reply docs/governance/technical_implementation/vote.md:116]: https://v3.squads.so/connect-squad

File: docs/governance/technical_implementation/vote.md [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+> In the following example, the Squad dashboard shows a Vault with 5 owners.
+> <img src={squads5owners} width="600" /> 
+> Clicking on the Vault drills the dash down to show the details for this Vault. 
+> <img src={vaultDash} width="600" /> 
+
+:::
+
+Find out more about Vaults in the [Squads documentation](https://docs.squads.so/squads-v3-docs/navigating-your-squad/transactions). Take some time to familiarize yourself with this interface.
+
+## Step 5: Transfer assets using MultiSig on Squads
```
Comment (m4sterbunny) [inline docs/governance/technical_implementation/vote.md:140]: Need to split into 2 paths: 1 account owner and 2 delegated account

Comment (m4sterbunny) [reply docs/governance/technical_implementation/vote.md:140]: Only creating accounts is different

**Review by YYYurchenko — APPROVED** [review]

**Pull request merged**